scripts/install_chart.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# Run from main directory
+DOCKER_TAG=${DOCKER_TAG:-"latest"}
+CHART_DIR="$(pwd)/chart/monarchs"
+echo "Using chart at $CHART_DIR" && test -d "$CHART_DIR"
+
+helm upgrade --install kingdb \
```

Comment:
Maybe this name should be consistent with the rest of the project?

Reply:
Oops, replaced it in Makefile and forgot to commit